targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K82F/drivers/fsl_flash.c

```diff
@@ -2734,12 +2734,13 @@ void flash_cache_clear(flash_config_t *config)
     __DSB();
 #endif /* FLASH_DRIVER_IS_FLASH_RESIDENT */
 }
-#if (defined(__CC_ARM))
-#pragma pop
-#endif
 #if (defined(__GNUC__))
 /* #pragma GCC pop_options */
-#endif
+#else
+#if (defined(__CC_ARM))
+#pragma pop
+#endif // (defined(__CC_ARM))
```

Comment:
consistency, they use `/* */` for comments, and I assume this is for readability, but not used in this file as it is on this line (no comments at endif, only for features).